### PR TITLE
Reduce redundancy for applying Icon in Button

### PR DESCRIPTION
**Pull request by itaditya**
In current code, Icon and Button are being listed two times each. If we need to alter one thing in say the Icon then I need to make changes at two places. This is a pain. My PR intends to fix that

### CODE REVIEW

File: src/components/atoms/button/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import React, { Fragment } from 'react'
```
Comment (siddharthkp) [inline src/components/atoms/button/index.js]: We can't use fragments because we want to support React 15 as well 😄 (without polyfills if we can help it)

**Comment by siddharthkp** [comment]
Thanks for taking the time!

File: src/components/atoms/button/index.js (hunk 1 above)
Comment (itaditya) [reply src/components/atoms/button/index.js]: @siddharthkp, I did a little bit of googling and I found that the array syntax you are currently using is supported with React 16 onwards. See https://reactjs.org/blog/2017/09/26/react-v16.0.html#new-render-return-types-fragments-and-strings

The `Fragment` feature is added in React 16.2 .

**Comment by itaditya** [comment]
Hey, what is the direction you are headed? Remove fragments to support React 15 or upgrade to react 16 ?

**Comment by siddharthkp** [comment]
Still difficult to answer, but tending towards 16

**Comment by nkohari** [comment]
Closing this for now since it's been open for awhile with no action. We can revisit this if needed.

**Pull request closed without merging**